data/pc/common/features.json

```diff
@@ -611,6 +524,11 @@
       }
     ]
   },
+  {
+    "name": "booksUseStoredEnchantments",
```

Comment:
This is because enchanted books don't use the StoredEnchantments tag on bedrock

Reply:
Why are you updating the pc data?

Reply:
because it makes more sense to have a feature that says StoredEnchantments is supported than to have one that says StoredEnchantments are not supported
StoredEnchantments is not used on bedrock